### ISSUE TITLE
Change the stylesheet to make the HTML page more readable.

### ISSUE THREAD

**Issue by NeilAn99** [NeilAn99/nan1-ssg#8]
The current colour contrast from foreground to background makes the text hard to read. Also, the text should be left aligned instead of centre aligned.
